os/arch/boardctl.c

```diff
@@ -150,7 +150,10 @@ int boardctl(unsigned int cmd, uintptr_t arg)
 		sched_lock();
 		/* Add 100ms delay for flushing another logs like printf. */
 		up_mdelay(100);
+#ifndef CONFIG_ARCH_BOARD_RTL8730E
```

Comment:
Could you let me know detail?

Reply:
This change is not required post https://github.com/Samsung/TizenRT/pull/5868